src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
```

Comment:
There should be a loading indicator when votes are loading after I open the modal

Reply:
https://www.loom.com/share/e4e0b2906e96458b9eebe942c8ceaf61